### PR TITLE
Dont fail when rendering UTFGrid with outline

### PR DESCRIPTION
**Pull request by charles-plante**
Add support for outline with a case test

### CODE REVIEW

**Comment by rouault** [comment]
Travis-CI failures seem unrelated. I've created https://trac.osgeo.org/osgeo/ticket/2381

**Comment by jmckenna** [comment]
I have restarted the demo server.

File: maputfgrid.cpp [hunk 1]
```diff
@@ -667,7 +667,11 @@ int utfgridRenderLine(imageObj *img, shapeObj *lineshape, strokeStyleObj *linest
 
   /* utfvalue is set to 0 if the shape isn't in the table. */
   if(r->utfvalue == 0) {
-    return MS_FAILURE;
+    //return MS_FAILURE;
```
Comment (dmorissette) [inline maputfgrid.cpp]: Why not delete this commented return statement?

Comment (rouault) [inline maputfgrid.cpp]:
```suggestion
```
Remove the commented line

**Pull request merged**